plv8.cc

```diff
@@ -1862,26 +1944,39 @@ GetGlobalObjectTemplate(Isolate *isolate)
 
 		Local<ObjectTemplate> templ = ObjectTemplate::New(isolate);
 		// ERROR levels for elog
-		templ->Set(String::NewFromUtf8(isolate, "DEBUG5", String::kInternalizedString), Int32::New(isolate, DEBUG5));
-		templ->Set(String::NewFromUtf8(isolate, "DEBUG4", String::kInternalizedString), Int32::New(isolate, DEBUG4));
-		templ->Set(String::NewFromUtf8(isolate, "DEBUG3", String::kInternalizedString), Int32::New(isolate, DEBUG3));
-		templ->Set(String::NewFromUtf8(isolate, "DEBUG2", String::kInternalizedString), Int32::New(isolate, DEBUG2));
-		templ->Set(String::NewFromUtf8(isolate, "DEBUG1", String::kInternalizedString), Int32::New(isolate, DEBUG1));
-		templ->Set(String::NewFromUtf8(isolate, "DEBUG", String::kInternalizedString), Int32::New(isolate, DEBUG5));
-		templ->Set(String::NewFromUtf8(isolate, "LOG", String::kInternalizedString), Int32::New(isolate, LOG));
-		templ->Set(String::NewFromUtf8(isolate, "INFO", String::kInternalizedString), Int32::New(isolate, INFO));
-		templ->Set(String::NewFromUtf8(isolate, "NOTICE", String::kInternalizedString), Int32::New(isolate, NOTICE));
-		templ->Set(String::NewFromUtf8(isolate, "WARNING", String::kInternalizedString), Int32::New(isolate, WARNING));
-		templ->Set(String::NewFromUtf8(isolate, "ERROR", String::kInternalizedString), Int32::New(isolate, ERROR));
+		templ->Set(String::NewFromUtf8(isolate, "DEBUG5", NewStringType::kInternalized).ToLocalChecked(),
+				   Int32::New(isolate, DEBUG5));
+		templ->Set(String::NewFromUtf8(isolate, "DEBUG4", NewStringType::kInternalized).ToLocalChecked(),
+				   Int32::New(isolate, DEBUG4));
+		templ->Set(String::NewFromUtf8(isolate, "DEBUG3", NewStringType::kInternalized).ToLocalChecked(),
+				   Int32::New(isolate, DEBUG3));
+		templ->Set(String::NewFromUtf8(isolate, "DEBUG2", NewStringType::kInternalized).ToLocalChecked(),
+				   Int32::New(isolate, DEBUG2));
+		templ->Set(String::NewFromUtf8(isolate, "DEBUG1", NewStringType::kInternalized).ToLocalChecked(),
+				   Int32::New(isolate, DEBUG1));
+		templ->Set(String::NewFromUtf8(isolate, "DEBUG", NewStringType::kInternalized).ToLocalChecked(),
+				   Int32::New(isolate, DEBUG5));
+		templ->Set(String::NewFromUtf8(isolate, "LOG", NewStringType::kInternalized).ToLocalChecked(),
+				   Int32::New(isolate, LOG));
+		templ->Set(String::NewFromUtf8(isolate, "INFO", NewStringType::kInternalized).ToLocalChecked(),
+				   Int32::New(isolate, INFO));
+		templ->Set(String::NewFromUtf8(isolate, "NOTICE", NewStringType::kInternalized).ToLocalChecked(),
+				   Int32::New(isolate, NOTICE));
+		templ->Set(String::NewFromUtf8(isolate, "WARNING", NewStringType::kInternalized).ToLocalChecked(),
+				   Int32::New(isolate, WARNING));
+		templ->Set(String::NewFromUtf8(isolate, "ERROR", NewStringType::kInternalized).ToLocalChecked(),
+				   Int32::New(isolate, ERROR));
```

Comment:
I think the 1000's of changes to `ToLocalChecked()` were the most frustrating to me - see my personal fork.